--- configure.in.OSmanOS	2012-11-28 10:29:53.000000000 +0200
+++ configure.in		2012-12-27 22:35:58.188643705 +0200
@@ -6205,7 +6205,9 @@
 if test "$with_system_clucene" = "yes"; then
     AC_MSG_RESULT([external])
     SYSTEM_CLUCENE=YES
-    PKG_CHECK_MODULES(CLUCENE, libclucene-core)
+    PKG_CHECK_MODULES(CLUCENE, libclucene-core)    
+    # hack for include in multiarch path...
+    CLUCENE_CFLAGS="$CLUCENE_CFLAGS -I/usr/lib/`dpkg-architecture -qDEB_HOST_MULTIARCH`"
 
     AC_LANG_PUSH([C++])
     save_CXXFLAGS=$CXXFLAGS
